boards/common/nrf52/Makefile.features

```diff
@@ -0,0 +1,10 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
we might want to move this line into the `cpu` code, possibly into `nrf5x_common/Makefile.features, as all nrf CPUs support GPIO and the feature does not require any configuration by the boards.

Maybe this is something we can do in a follow-up PR, also taking the nrf51-based boards into account?!

Reply:
Apparently, there is no consensus if these should go in cpu or boards. See #10434 where I tried to move these definitions to cpu for sam0 based boards.
I'm not sure what's best.

Reply:
see my comment (and my ACK) in #10434 :-)

Reply:
if you want, you/we can address this in a separate/follow-up PR, so this PR can be merged without waiting for this discussion.

Reply:
ok, I missed that comment, so I'll have to remove the latest changes.